doc/source/conf.py

```diff
@@ -14,13 +14,8 @@
 # import sys
 
 import datetime
-import subprocess
-from collections import OrderedDict
-
 import MDAnalysis as mda
```

Comment:
There's some pre-commit failure with import orders and an extra blank line on 124 if that matters to you.